src/explore-education-statistics-common/src/modules/table-tool/components/DownloadCsvButton.tsx

```diff
@@ -10,6 +10,7 @@ import cartesian from '@common/utils/cartesian';
 import camelCase from 'lodash/camelCase';
 import React from 'react';
 import { utils, writeFile } from 'xlsx';
+import { logEvent } from '../../../../../explore-education-statistics-frontend/src/services/googleAnalyticsService';
```

Comment:
You can't import GA from the frontend into common, it will break the admin app. We need to make these components more extensible  to allow us to make the `logEvent` calls in frontend-only components.